src/EnergyPlus/InputProcessing/InputProcessor.hh

```diff
@@ -137,6 +137,8 @@ public:
 
     std::pair<std::string, bool> getObjectItemValue(std::string const &field_value, json const &schema_field_obj);
 
+    const json &getObjectItem(EnergyPlusData &state, std::string_view ObjType, std::string_view ObjName);
```

Comment:
new function in input processor to return a **json** object given it's type and name.